fastpack/cli.py

```diff
@@ -105,3 +112,7 @@ def main():
     # Make initial commit
     subprocess.run(["git", "add", "."], check=True)
     subprocess.run(["git", "commit", "-m", "Initial commit"], check=True)
+
+
+if __name__ == "__main__":
```

Comment:
Not for now, but something we should look into in the future:

checkout `click` for CLI tools.  I started using it recently in other packagess